Add missing zlib links.
```
/usr/lib/gcc/x86_64-pc-linux-gnu/15/../../../../x86_64-pc-linux-gnu/bin/ld: src/MEGASync/mega/tools/gfxworker/liblibgfxworker.a(logger.cpp.o): undefined reference to symbol 'gzputs'
```
--- a/src/MEGASync/control/control.cmake
+++ b/src/MEGASync/control/control.cmake
@@ -71,6 +71,9 @@ set(DESKTOP_APP_CONTROL_SOURCES
     control/StatsEventHandler.cpp
 )
 
+find_package(ZLIB REQUIRED)
+target_link_libraries(MEGAsync PRIVATE ZLIB::ZLIB)
+
 target_sources(MEGAsync
     PRIVATE
     ${DESKTOP_APP_CONTROL_HEADERS}
--- a/src/MEGASync/mega/tools/gfxworker/CMakeLists.txt
+++ b/src/MEGASync/mega/tools/gfxworker/CMakeLists.txt
@@ -70,9 +70,11 @@ add_executable(gfxworker
     src/main.cpp
 )
 
+find_package(ZLIB REQUIRED)
 target_link_libraries(gfxworker
     PRIVATE
     libgfxworker
+    ZLIB::ZLIB
 )
 
 # Adjust compilation flags for warnings and errors for gfxworker
